### PR TITLE
Release 0.0.4

### PR DESCRIPTION
**Pull request by stolarczyk**
This contribution makes changes to the `Project` object initialization to match the overall concept (the subproject can be specified) and adds backward compatibility to the `implied/derived_columns` (`implied/derived_attributes` now)

### CODE REVIEW

**Comment by nsheff** [comment]
wait -- why do we need a `subproject` slot on the `Project` object? I don't follow...

**Comment by stolarczyk** [comment]
You're right, we don't need it. It was the easiest way to pass the subproject name to the `initialize()` method and that's why I did it. Fixed it in: https://github.com/pepkit/pepr/commit/5612340db613c7537b9df2de047afae5e4d5b358

File: R/loadConfig.R [hunk 1]
```diff
@@ -69,55 +71,17 @@ loadConfig = function(filename=NULL, sp=NULL) {
...
+    invisible(names(cfg$subprojects))
+  }else{
+    # Otherwise return NULL for testing purposes
+    invisible(NULL)
   }
-  invisible(names(cfg$subprojects))
 }
 
 
-#' Mapper of organism name to genomic assembly name
```
Comment (nsheff) [inline R/loadConfig.R:79]: What were these assembly by organism functions written for? was this written by @vreuter ?

Comment (vreuter) [reply R/loadConfig.R:79]: I wrote something similar to this for `peppy`/`looper` but don't remember doing so for this package

Comment (stolarczyk) [reply R/loadConfig.R:79]: It wasn't me either

**Review by nsheff — CHANGES_REQUESTED** [review]
Please keep line length within bioc specificiations

File: R/loadConfig.R [hunk 2]
```diff
@@ -69,88 +71,69 @@ loadConfig = function(filename=NULL, sp=NULL) {
...
-#' an \code{implied_columns} section.
-#'
-#' @param config A project configuration object (parsed from a file with 
-#'        \code{yaml::yaml.load_file}).
-#' @return Mapping (as \code{list}) in which each name is an organism, and 
-#'         each list element is the corresponding genomic assembly name.
-#' @seealso \url{http://looper.readthedocs.io/en/latest/implied-columns.html}
+#' Expand system path
+#' 
+#' This function expands system paths (the non-absolute paths become absolute) and replaces the enviornment variables (e.g, \code{${HOME}}) with their vaules.
```
Comment (nsheff) [inline R/loadConfig.R]: spelling check here, environment; values -- and line length. if you use an IDE that  that can wrap lines that can help

**Comment by nsheff** [comment]
Looking good @michalstolarczyk  -- are you ready to merge this (after my minor style change suggestions)?

**Comment by nsheff** [comment]
Can you make sure to pass `R CMD check` on this?

**Comment by stolarczyk** [comment]
Ok, made the changes to the code, package passes the `R CMD check`. Let's merge

**Comment by nsheff** [comment]
Don't put the pkgdown site in the package repository; we host them at code.databio.org

**Review by nsheff — APPROVED** [review]

**Pull request merged**